project/app/public/stylesheets/style.css

```diff
@@ -9,4 +11,38 @@ a {
 
 input:invalid {
   border: 1px dashed red;
+  width: 25%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+}
+
+input {
+  width: 25%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+}
+
+.NavButton{
+  background-color: white;
+  color: black;
+  padding: 15px 32px;
+  text-align: center;
+  display: inline-block;
+  font-size: 16px;
+  position:relative;
+  top:40%;
+  left:0%;
+  cursor:pointer;
+  transition: 0.3s;
+  opacity: 0.6;
+}
+.NavButton:hover {opacity: 1;}
+
+.TitleImage {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```

Comment:
instead of adjusting width and height of individual element, a more scalable way is to confine elements into rows and columns, check how @dccxi did it by using classes like `row`, `col-1` `col-md-1` etc.